src/Core/src/Fonts/FontRegistrar.cs

```diff
@@ -50,7 +50,7 @@ public void Register(string filename, string? alias)
 		/// <inheritdoc/>
 		public string? GetFont(string font)
 		{
-			if (_fontLookupCache.TryGetValue(font, out var foundResult))
+			if (_fontLookupCache.TryGetValue(font, out var foundResult) && File.Exists(foundResult))
```

Comment:
Thanks @mattleibow!

The font file on Android might disappear because we're copying it to the temporary file path. This location can be cleared by the OS (or user) at any time, even while the app is running. I don't think this issue applies to other platforms (and I don't think my original changes would have worked cross-platform anyway!), so I've  I've pushed some changes that I think will only apply the file check on Android. Let me know if there's a better way to do this!

I've added an IsCachedFontValid method with a per-platform implementation that just returns true for everything, but on Android it checks if the file exists.

I had to remove my original test as it wasn't platform specific, so now I've tried to add an Android-only test, but I've not been able to get it to run locally yet, still working that out!

Reply:
@mattleibow just wondering if you’d had a chance to review my changes and the comments above? Thanks.